CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 # NMOS Common Library Changelog
 
+## 0.17.3
```

Comment:
This probably ought to be a minor version bump rather than patch given it adds a feature